tests/test_core.py

```diff
@@ -185,12 +210,24 @@ def test_time_function_decorator(self):
 
         @self.gauge.time()
         def f():
-            time.sleep(.001)
+            time.sleep(.05)
 
         self.assertEqual(([], None, None, None), getargspec(f))
 
         f()
-        self.assertNotEqual(0, self.registry.get_sample_value('g'))
+        self.assertTrue(0.05 <= self.registry.get_sample_value('g') <= 0.1)
```

Comment:
It is better to check the expected duration. It makes more sense in a case of async functions.